modules/ballerina-test/src/test/java/org/ballerinalang/test/services/testutils/MessageUtils.java

```diff
@@ -141,4 +151,30 @@ private static CarbonMessage setWebSocketCommonProperties(CarbonMessage carbonMe
                                   new LinkedList<>());
         return carbonMessage;
     }
+
+    /**
+     * Get the response value from input stream.
+     *
+     * @param response carbon response
+     * @return return value from  input stream as a string
+     */
+    public static String getReturnValue(HTTPCarbonMessage response) {
+        Reader reader;
+        final int bufferSize = 1024;
+        final char[] buffer = new char[bufferSize];
+        final StringBuilder out = new StringBuilder();
+        try {
+            reader = new InputStreamReader(new HttpMessageDataStreamer(response).getInputStream(), UTF_8);
+            while (true) {
+                int size = reader.read(buffer, 0, buffer.length);
+                if (size < 0) {
+                    break;
+                }
+                out.append(buffer, 0, size);
+            }
+        } catch (IOException e) {
+            LOG.error("Error occured while reading the response value in getReturnValue", e.getMessage());
+        }
+        return out.toString();
+    }
```

Comment:
The same functionality is duplicated in ResponseReader.java. Shall we remove one of them and use same in all the places?